fix(lldap) fix password requirements with the correct validation type.

neotoxicfr opened the pull request:
**⚒️ Fixes**

Change min value for LLDAP password to min length

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [x] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**✔️ Checklist:**

- [x] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [x] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [x] ⬆️ I increased versions for any altered app according to semantic versioning

neotoxicfr committed:
Fix LLDAP password requirements

Change min value to min length

Signed-off-by: NeoToxic <remidu34070@hotmail.fr>

neotoxicfr committed:
Update Chart.yaml

Signed-off-by: NeoToxic <remidu34070@hotmail.fr>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/truecharts/charts?pullRequest=10774) <br/>All committers have signed the CLA.

stavros-k commented:
@xstar97 Don't enable auto-merge without CLA singed please.
@neoestremi Please don't blindly tick checkboxes for things that have not been introduced in this PR :) 
also Please sign CLA

neotoxicfr commented:
Done, sorry, I thought I could check boxes if I wasn't concerned but the app was already fonctionnal.
Signed the CLA.

stavros-k commented on charts/stable/lldap/questions.yaml:
```diff
@@ -41,7 +41,7 @@ questions:
                                           private: true
                                           default: ""
-                                          min: 8
+                                          minLength: 8
```
```suggestion
                                          max_length: 8
```

xstar97 replied:
max length? lldap has a min length requirement of 8....

stavros-k committed:
Update charts/stable/lldap/questions.yaml

Signed-off-by: Stavros Kois <47820033+stavros-k@users.noreply.github.com>

xstar97 commented on charts/stable/lldap/questions.yaml:
```diff
@@ -41,7 +41,7 @@ questions:
                                           private: true
                                           default: ""
-                                          min: 8
+                                          max_length: 8
```
shouldnt this be min_length?
sauce -> https://github.com/lldap/lldap/issues/521

xstar97 commented on charts/stable/lldap/questions.yaml:
```diff
@@ -41,7 +41,7 @@ questions:
                                           private: true
                                           default: ""
-                                          min: 8
+                                          max_length: 8
```
password policy is a min of 8....lets not cap the password to a _max_ of 8.
https://github.com/lldap/lldap/issues/521
```suggestion
                                          min_length: 8
```

stavros-k replied:
Yep bad copy paste

xstar97 committed:
Update charts/stable/lldap/questions.yaml

lldap checls for a password of a min length of 8, lets not default to a max of 8.

Signed-off-by: Xstar97TheNoob <9399967+xstar97@users.noreply.github.com>

truecharts-admin commented:
This PR is locked to prevent necro-posting on closed PRs. Please create a issue or contact staff on discord if you want to further discuss this
